frontend/payment-klarna/components/Checkout.vue

```diff
@@ -92,9 +92,10 @@ export default {
     resumeCheckout () {
       return callApi(api => api.resume())
     },
-    onKcoAddressChange () {
+    onKcoAddressChange (shippingAddress) {
       return callApi(api => api.on({
         'change': (data) => {
```

Comment:
Should we update `change` to `shipping_address_change` @sn00k ?

Reply:
Looks like when the address for the customer is preloaded, this event is not triggered. Seems weird but I can't register the call anyway. Will keep it as `change` for now.